odoo/addons/base/tests/test_translate.py

```diff
@@ -997,6 +997,20 @@ def test_sync_xml_no_en(self):
         self.assertEqual(view.with_env(env_fr).arch_db, archf % terms_fr)
         self.assertEqual(view.with_env(env_nl).arch_db, archf % terms_nl)
 
+    def test_sync_xml_attribute(self):
+        """ check translations with attribute can be cleaned up after write """
+        self.env['res.lang']._activate_lang('fr_FR')
+        archf = '<form><i title="%s"/></form>'
+        terms_en = ['Fork',]
+        terms_fr = ['Fourchetta',]
```

Comment:
```suggestion
        terms_en = ('Fork',)
        terms_fr = ('Fourchetta',)
```
Otherwise: `'<form><i title="[\'Fourchetta\']"/></form>'`